README.md

```diff
@@ -1,12 +1,9 @@
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)  [![Build Status](https://travis-ci.com/andela/phoenix-bn-backend.svg?branch=develop)](https://travis-ci.com/andela/phoenix-bn-backend)
-
-# Barefoot Nomad - Making company travel and accomodation easy and convinient.
+# Barefoot Nomad - Making company travel and accommodation easy and convenient
 
+[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)  [![Build Status](https://travis-ci.com/andela/phoenix-bn-backend.svg?branch=develop)](https://travis-ci.com/andela/phoenix-bn-backend) [![Coverage Status](https://coveralls.io/repos/github/andela/phoenix-bn-backend/badge.svg?branch=ch-coveralls-167765384)](https://coveralls.io/github/andela/phoenix-bn-backend?branch=ch-coveralls-167765384) [![Maintainability](https://api.codeclimate.com/v1/badges/28351a9e117de0ecbba0/maintainability)](https://codeclimate.com/github/andela/phoenix-bn-backend/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/28351a9e117de0ecbba0/test_coverage)](https://codeclimate.com/github/andela/phoenix-bn-backend/test_coverage)
 
```

Comment:
Did you integrate codeclimate as well?

Reply:
Yes, coveralls doesn't have maintainability metrics